scripts/us_bea/states_gdp/import_data.py

```diff
@@ -62,12 +62,16 @@ def __init__(self):
         self.raw_df = None
         self.clean_df = None
 
-    def download_data(self):
+    def download_data(self, file=None, zip_link=None):
```

Comment:
Per https://google.github.io/styleguide/pyguide.html#383-functions-and-methods, please add a function comment, especially defining what "file" and "zip_link" are supposed to be.